ioctl/cmd/config/configsetget.go

```diff
@@ -26,19 +26,22 @@ const (
 )
 
 var (
-	validArgs       = []string{"endpoint", "wallet", "currentcontext"}
+	validArgs       = []string{"endpoint", "wallet", "explorer", "defaultacc"}
+	validGetArgs    = []string{"endpoint", "wallet", "explorer", "defaultacc", "all"}
+	validExpl       = []string{"iotexscan", "iotxplorer"}
```

Comment:
Personally, I prefer ioctl supporting editable url rather than several options, which is more friendly to explorer developer.

Reply:
I added an option for custom link

Reply:
cool